src/js/stores/AppFormStore.js

```diff
@@ -388,13 +388,15 @@ function processResponseErrors(responseErrors, response, statusCode) {
   } else if (statusCode === 422 && response != null &&
       Util.isArray(response.details)) {
     response.details.forEach(detail => {
-      var attributePath = detail.attribute.length
-        ? detail.attribute
-        : "general";
-      var fieldId = resolveResponseAttributePathToFieldId(attributePath) ||
-        attributePath;
-      responseErrors[fieldId] =
-        AppFormErrorMessages.lookupServerResponseMessage(detail.error);
+      var path = detail.path;
+      var fieldId = resolveResponseAttributePathToFieldId(path) || path;
+      if (detail.errors.length >= 1) {
+        responseErrors[fieldId] =
+          AppFormErrorMessages.lookupServerResponseMessage(detail.errors[0]);
```

Comment:
Would it make sense to iterate through all `detail.errors` and return all errors as one concatenated string?